app/brave_settings_strings.grdp

```diff
@@ -25,6 +25,136 @@
   <message name="IDS_SETTINGS_BRAVE_GET_STARTED_TITLE" desc="The title for the Get Started section in settings">
     Get started
   </message>
+
+  <!-- <Brave Account> -->
+  <!-- Row: -->
+  <message name="IDS_SETTINGS_BRAVE_ACCOUNT_ROW_TITLE" translateable="false" desc="Title of the Brave Account row in the Get Started section">
+    Sign in or create a Brave account
+  </message>
+  <message name="IDS_SETTINGS_BRAVE_ACCOUNT_ROW_DESCRIPTION" translateable="false" desc="Description of the Brave Account row in the Get Started section">
+    A Brave account will allow you to easily sync your settings, passwords, bookmarks, etc. across all your devices and access our full range of premium products.
```

Comment:
We'll need to update that since we are shipping accounts without support for Sync (for the time being).